java/src/jmri/jmrit/blockboss/configurexml/BlockBossLogicProviderXml.java

```diff
@@ -183,7 +183,7 @@ public boolean load(Element shared, Element perNode) {
             try {
                 bb = getBlockBossLogicFromElement(block);
                 if (bb == null)
-                    break;
+                    continue;
```

Comment:
The xml load process has been updated to better handle exceptional conditions.

The change to **continue** is so that all valid SSL definitions are loaded instead of stopping with the first bad one.  This was causing people to have disappearing SSL definitions.